indispensable-josef/src/commonMain/kotlin/at/asitplus/signum/indispensable/josef/JsonWebToken.kt

```diff
@@ -48,10 +48,30 @@ data class JsonWebToken(
      * attestation.
      */
     @SerialName("cnf")
-    val confirmationKey: JsonWebKey? = null,
+    val confirmationClaim: ConfirmationClaim? = null,
+    /**
+     * RFC 9449: The value of the HTTP method (Section 9.1 of [RFC9110](https://datatracker.ietf.org/doc/html/rfc9110))
+     * of the request to which the JWT is attached.
+     */
+    @SerialName("htm")
+    val httpMethod: String? = null,
+    /**
+     * RFC 9449: The HTTP target URI (Section 7.1 of [RFC9110](https://datatracker.ietf.org/doc/html/rfc9110)) of the
+     * request to which the JWT is attached, without query and fragment parts.
+     */
+    @SerialName("htu")
+    val httpTargetUrl: String? = null,
```

Comment:
why not uri/url?

Reply:
Because there is no dependency on ktor

Reply:
oh. i was not aware that kotlin does not have it in the stdlib